scripts/ai/AIDriveStrategyCombineCourse.lua

```diff
@@ -1307,7 +1307,7 @@ end
 
 --- Can the cutter be turned off ?
 function AIDriveStrategyCombineCourse:getCanCutterBeTurnedOff()
-    return self:isWaitingForUnload() or
+    return self:isWaitingForUnload() or self.state == self.states.TURNING or

```

Comment:
This change would most likely turn off the cutters for normal combines making a headland turn with/without a pocket.

Reply:
True. Good catch. I will test this throughoutly. First intuition would be to make row changing turn it's own state, but I need to check how much that would affect other code.